pandas/tests/indexing/test_indexing.py

```diff
@@ -225,6 +221,10 @@ def test_dups_fancy_indexing(self):
             result = df.loc[['A', 'A', 'E']]
         tm.assert_frame_equal(result, expected)
 
+        if PY2:
```

Comment:
can you split this test, and use the skip decorator instead (and add the reason)